spanner/api/Program.cs

```diff
@@ -0,0 +1,989 @@
+                cmd.Parameters["AlbumTitle"].Value = "Good Hurrian Songs";
+                await cmd.ExecuteNonQueryAsync();
+                // Insert second row.
+                cmd.Parameters["SingerId"].Value = 1;
+                cmd.Parameters["AlbumId"].Value = 2;
+                cmd.Parameters["AlbumTitle"].Value = "Cuneiform Chorus";
+                await cmd.ExecuteNonQueryAsync();
+                // Insert third row.
+                cmd.Parameters["SingerId"].Value = 2;
+                cmd.Parameters["AlbumId"].Value = 1;
+                cmd.Parameters["AlbumTitle"].Value = "Zany Mbira Music";
+                await cmd.ExecuteNonQueryAsync();
+                // Insert fourth row.
+                cmd.Parameters["SingerId"].Value = 2;
+                cmd.Parameters["AlbumId"].Value = 2;
+                cmd.Parameters["AlbumTitle"].Value = "Ambient Sistrum Sirens";
+                await cmd.ExecuteNonQueryAsync();
+                // Insert fifth row.
+                cmd.Parameters["SingerId"].Value = 2;
+                cmd.Parameters["AlbumId"].Value = 3;
+                cmd.Parameters["AlbumTitle"].Value = "Congo Carols";
+                await cmd.ExecuteNonQueryAsync();
+                Console.WriteLine("Inserted data.");
+                return;
+            }
+            // [END insert_data]
+        }
+
+        public static async Task CreateDatabaseAsync(
+            string projectId, string instanceId, string databaseId)
+        {
+            // [START create_custom_database]
+            // Initialize request connection string for database creation.
+            string connectionString =
+                $"Data Source=projects/{projectId}/instances/{instanceId}";
+            // Make the request.
+            using (var connection = new SpannerConnection(connectionString))
+            {
+                string createStatement = $"CREATE DATABASE `{databaseId}`";
+                var cmd = connection.CreateDdlCommand(createStatement);
+                await cmd.ExecuteNonQueryAsync();
+            }
+            return;
+            // [END create_custom_database]
+        }
+
+        public static object CreateSampleDatabase(string projectId,
+    string instanceId, string databaseId)
+        {
+            var response = Task.Run(async () =>
+                await CreateSampleDatabaseAsync(
+                    projectId, instanceId, databaseId));
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);
+            Console.WriteLine($"Operation status: {response.Status}");
+            Console.WriteLine($"Created sample database {databaseId} on "
+                + $"instance {instanceId}");
+            return 0;
+        }
+
+        public static object InsertSampleData(string projectId,
+string instanceId, string databaseId)
+        {
+            var response = Task.Run(async () => await InsertSampleDataAsync(

```

Comment:
You don't need `async/await` in cases like this when when the only task to await is a return at the end.